lib/rules/font-family-name-quotes/index.js

```diff
@@ -56,8 +56,67 @@ function quotesRequired(family) {
 		.some((word) => /^(?:-?\d|--)/.test(word) || !/^[-\w\u{00A0}-\u{10FFFF}]+$/u.test(word));
 }
 
+/**
+ * @typedef {{
+ * name: string,
+ * hasQuotes: boolean,
+ * sourceIndex: number,
+ * removeQuotes: function,
+ * addQuotes: function
+ * }} MutableNode
+ */
+
+/**
+ *
+ * @param {import('postcss-value-parser').Node[]} fontFamilies
+ * @param {import('postcss').Declaration} decl
```

Comment:
[suggest] How about adding `@returns` for more readability and type-safety?

```suggestion
 * @param {import('postcss').Declaration} decl
 * @returns {MutableNode[]}
```